### PR TITLE
Add template_uri & stylesheet_uri

### PR DESCRIPTION
**Pull request by tditlu**
This will mimic the helpers template_path & stylesheet_path, but for URL's.

### CODE REVIEW

File: src/helpers.php [hunk 1]
```diff
@@ -123,6 +109,22 @@ function stylesheet_path(string $path = ''): string
...
+    /**
+     * Generate a uri for the current/child theme directory.
+     *
+     * @param string $path
+     *
+     * @return string
+     */
+    function stylesheet_uri(string $path = ''): string
+    {
+        $path = $path ? '/'.$path : $path;
```
Comment (vinkla) [inline src/helpers.php]: @tditlu how does this differ from the previous setup using `ltrim`?

**Comment by vinkla** [comment]
Thanks for the pull request!

What is the use case here? Isn't this already possible with `asset`?

Couple of things we need to update before we can merge this pull request:

- Add tests for the new helper functions
- Order the functions in alphabetical order

**Comment by tditlu** [comment]
@vinkla: Tests added & functions are now ordered alphabetically

**Comment by vinkla** [comment]
Merged!

**Pull request merged**